libcxx/test/std/numerics/rand/rand.eng/rand.eng.lcong/assign.pass.cpp

```diff
@@ -61,24 +61,33 @@ test()
   test1<T, A, 0, M>();
   test1<T, A, M - 2, M>();
   test1<T, A, M - 1, M>();
+}
+
+template <class T>
+void test_ext() {
+  const T M(static_cast<T>(-1));
 
-  /*
```

Comment:
I assume the next line's " (not implemented)" is no longer true too. (The same for similar enabled tests).